File: python/fizz_buzz.py
def fizz_buzz(number):
    if number % 15 == 0:
        print('fizzbuzz')
        return 
    elif number % 5 == 0:
        print('fizz')
        return
    elif number % 3 == 0:
        print('buzz')
        return
    else:
        print('Number not fizz, buzz, or fizzbuzz')


if __name__ == "__main__":
    fizz_buzz(1)
    fizz_buzz(3)
    fizz_buzz(5)
    fizz_buzz(15)

File: javascript/fizzBuzz.js
let fizzBuzz = (number) => {
    if (number % 15 === 0) {
        console.log('fizzbuzz')
        return 
    }
    else if (number %5 ===0) {
        console.log('fizz')
        return
    }
    else if (number %3 ===0) {
        console.log('buzz')
        return
    }
    else {
        console.log(number)
        return
    }

} 


fizzBuzz(1)
fizzBuzz(3)
fizzBuzz(5)
fizzBuzz(15)
fizzBuzz(2)
// // ES6
// const multiplyES6 = (x, y) => { return x * y };